its/src/test/kotlin/org/sonarlint/intellij/its/utils/SettingsUtils.kt

```diff
@@ -81,6 +81,18 @@ class SettingsUtils {
             }
         }
 
+        fun toggleRule() {
```

Comment:
Would be nice if we could pass the rule key and the rule text as part of the method parameters